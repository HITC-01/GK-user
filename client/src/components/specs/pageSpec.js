import React from 'react';
import renderer from 'react-test-renderer';
import Page from '../Page.jsx';
import "isomorphic-fetch";

describe('<Page />', () => {
  test('Should render correctly', () => {
    const component = renderer.create(<Page />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
