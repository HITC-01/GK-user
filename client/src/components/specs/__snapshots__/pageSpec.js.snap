// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> Should render correctly 1`] = `
<div
  className="up-app"
>
  <div
    className="up"
  >
    <img
      alt="Loading"
      className="up-photo"
    />
    <br />
    <span
      className="up-username"
    />
    <span
      className="up-bolt"
    >
      
    </span>
    <br />
    <div
      className="up-count"
    >
      <i
        className="fas fa-users"
      />
      <span
        className="up-followcount up-txt"
      />
      <i
        className="fas fa-headphones-alt"
      />
      <span
        className="up-trackcount up-txt"
      />
    </div>
    <br />
    <button
      className="up-button-following"
      onClick={[Function]}
      tabIndex="0"
      title="Following"
      type="button"
    >
      following
    </button>
    <br />
    <div
      id="up-report-sign"
    >
      <i
        className="far fa-flag"
      />
      <button
        className="up-report-button up-txt"
        onClick={[Function]}
        type="button"
      >
        Report
      </button>
      <div
        className="up-modalWrapper up-display-none"
      >
        <div
          className="up-modal"
        >
          <section
            className="up-modal-report"
          >
            <h1>
               Report Track For 
            </h1>
            <br />
            <ul>
              <li>
                <a
                  href="www.google.com"
                >
                   Copyright infringement 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Privacy Violation 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Pornographic Content 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Abuse 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Hate Speech 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Illegal Content 
                </a>
              </li>
              <br />
              <li>
                <a
                  href="www.google.com"
                >
                   Other 
                </a>
              </li>
            </ul>
          </section>
          <button
            className="up-close"
            onClick={[Function]}
            type="button"
          >
             X 
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="up-td-div"
  >
    <p
      className="up-trackdescription"
    >
      
    </p>
    <button
      className="up-bt-show-more"
      type="button"
    >
       Show more 
    </button>
  </div>
</div>
`;
